src/endpoints/campaigns/get_free_domain.rs

```diff
@@ -35,6 +35,8 @@ pub async fn handler(
     State(state): State<Arc<AppState>>,
     Query(query): Query<FreeDomainQuery>,
 ) -> impl IntoResponse {
+    let conf = config::load();
+    let logger = Logger::new(&conf.watchtower);
```

Comment:
We don't want to have multiple logger instance. You can use state.logger instead (which returns the logger instance created in main.rs)